pandas/plotting/_core.py

```diff
@@ -833,6 +833,23 @@ def _post_plot_logic(self, ax, data):
         ax.set_ylabel(pprint_thing(y))
         ax.set_xlabel(pprint_thing(x))
 
+    def _plot_colorbar(self, kws):
+        # In IPython inline backend the colorbar axis height
+        # tends to not exactly match the parent axis height.
+        # The difference is due to small fractional differences
+        # in floating points with similar representation.
+        # To deal with this, this method forces the colorbar
+        # height to take the height of the parent axes.
+        ax = kws['ax']
+        img = ax.collections[0]
```

Comment:
is there an issue we can reference about this impl. IOW next person to look at wouldn't know where these set_position come from. so a) ref this issue in a comment and b) an Ipython refernce (if you can)